src/screens/Followings/FollowersScreen.js

```diff
@@ -88,6 +95,10 @@ function FollowersScreen() {
       unsubFocusListener();
     };
   }, []);
+
+  const followedUsers = dataFollower.filter((item) => item.user.following);
+  const unfollowedUsers = dataFollower.filter((item) => !item.user.following);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The follower data (`dataFollower`) is now being filtered into two categories: `followedUsers` and `unfollowedUsers`. This seems like a good approach to separate the followers based on whether they are followed by the current user or not.
```javascript
const followedUsers = dataFollower.filter((item) => item.user.following);
const unfollowedUsers = dataFollower.filter((item) => !item.user.following);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->